pgmpy/tests/test_base/test_MAG.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+
+from pgmpy.base import MAG
+
+
+@pytest.fixture
```

Comment:
We need to clean up the tests. Try to keep one test function for each method in the class. This file is unreadable right now.